docs/code-samples/getting-started/README.md

```diff
@@ -1,10 +1,12 @@
 # Code Samples - Getting Started
 
-Note - This project is currently an Early Access 'Alpha' feature (see [here](https://docs.daml.com/support/status-definitions.html) for more information).
+Note - This project is currently an Early Access 'Alpha' feature (see
```

Comment:
We need to update this as `Daml-Finance` will move to GA after this release. You can remove this line.

Reply:
removed the early access line. we need to update the `Package Status` section in a follow up PR as well.